### PR TITLE
Adjust permissions

### LINKED ISSUE #52: Adjust permissions
Permission u_see_voters is global, cannot be set by forum.  It should be possible to have this extra granularity.  So, we should remove u_see_voters permission, and replace it with local permission f_seevoters, make appropriate checks and changes to code, and register a new event to properly classify this permission as a "Forum/Poll" category permission.

### PR DESCRIPTION
**Pull request by javiexin**
Allows permissions to see poll voters to be set by forum.
Closes #52

### CODE REVIEW

File: migrations/v1_2_2_data_permissions.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * Advanced Polls
+ *
+ * @copyright (c) 2015 Wolfsblvt ( www.pinkes-forum.de )
+ * @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
+ * @author Clemens Husung (Wolfsblvt)
+ */
+
+namespace wolfsblvt\advancedpolls\migrations;
+
+class v1_2_2_data_permissions extends \phpbb\db\migration\migration
```
Comment (Wolfsblvt) [inline migrations/v1_2_2_data_permissions.php]: We are still working vor version 1.2.0, so can you rename the file to `v1_2_0_data_permissions`?

File: event/listener.php [hunk 2]
```diff
@@ -62,6 +63,20 @@ public static function getSubscribedEvents()
 	}
 
 	/**
+	 * Adds the permission to the right permission category
+	 *
+	 * @param object $event The event object
+	 * @return void
+	 */
+	public function adv_polls_permissions($event)
```
Comment (Wolfsblvt) [inline event/listener.php]: I am not quite informed, so just a question:
Why is this method needed? To add the permission to the right category? Isn't it be already added to some category by default?

Comment (javiexin) [inline event/listener.php]: It is added to the "Misc" category, and cannot be changed any other way than by subscribing to this event...
This is something that should be stored in the DB, and changed in migration files, but... not there yet :-1:

Comment (Wolfsblvt) [inline event/listener.php]: okay, good.

**Pull request merged**